server/utils/utils.ts

```diff
@@ -77,3 +77,20 @@ export const convertDateAndTimeInputsToIsoString = <K extends string | number>(
 
   return dateInputObj
 }
+
+export const dateAndTimeInputsAreValidDates = <K extends string | number>(
+  dateInputObj: ObjectWithDateParts<K>,
+  key: K,
+): boolean => {
+  const dateString = convertDateAndTimeInputsToIsoString(dateInputObj, key)
+
+  try {
+    convertDateString(dateString[key])
```

Comment:
Not for now but it would be good to know what validation is currently in place in Delius